### ISSUE TITLE
initialize

### ISSUE THREAD

**Issue by harig00** [AlainPlattner/Slepian#3]
When we set the path to IFILES, shouldn't it be the fullpath and not a relative path? 

I had a student create another directory where they will do their project work, but as it stands you are required to run initialize.m in the base Slepian directory. This might be more tricky to make work, but it would be more flexible.

Thoughts?

**Comment by AlainPlattner** [comment]
Good point. I had been thinking that relative paths make the installation easiest, because the installation script does not need to know about the absolute path, where the user wants to install the software.

Another option would be to create a string in the initialize script that contains the absolute path of the Slepian folder and then just prepend this string to all the paths. This would allow users who want to use the relative paths to just leave the script as it is, and those who want to use absolute paths could just edit that string. 

What do you think about this idea?
